distributed/deploy/adaptive_core.py

```diff
@@ -78,10 +78,13 @@ def __init__(
         self.periodic_callback = None
 
         def f():
-            self.periodic_callback = PeriodicCallback(self.adapt, self.interval * 1000)
-            self.periodic_callback.start()
+            try:
+                self.periodic_callback.start()
+            except AttributeError:
+                pass
 
         if self.interval:
+            self.periodic_callback = PeriodicCallback(self.adapt, self.interval * 1000)
```

Comment:
Do we ever call this from a thread that isn't the IOLoop?  If so, does this pick up the IOLoop at instantiation time or at start time?  Usually when we use `add_callback` it's because we're handling something that is sensitive to where it gets started and we're not always on the event loop.

Reply:
The loop is picked up at start time. 

https://www.tornadoweb.org/en/stable/_modules/tornado/ioloop.html#PeriodicCallback.start

There is nothing in the `PeriodicCallback.__init__` that makes me think this would be a problem.

Reply:
Oh awesome.  Merging then.